### PR TITLE
🌐Updated language files

### PR DESCRIPTION
**Pull request by MrKrisKrisu**

### CODE REVIEW

File: resources/lang/en.json [hunk 1]
```diff
@@ -368,5 +368,11 @@
...
     "transport_types.suburban": "suburban",
     "transport_types.subway": "subway",
-    "transport_types.tram": "tram"
+    "transport_types.tram": "tram",
+    "dateformat.with-weekday": "dddd, DD. MMMM YYYY",
+    "leaderboard.month.title": "Monthly overview",
+    "time-format": "hh:mm a",
+    "datetime-format": "DD.MM.YYYY HH:mm a",
+    "dateformat.month-and-year": "MMMM YYYY",
+    "time-format.with-day": "hh:mm a (DD.MM.YYYY)"
```
Comment (HerrLevin) [inline resources/lang/en.json:377]: shouldn't this be HH?

Comment (HerrLevin) [reply resources/lang/en.json:377]: or is that 12h?

**Review by HerrLevin — CHANGES_REQUESTED** [review]

File: resources/lang/en.json (hunk 1 above)
Comment (MrKrisKrisu) [reply resources/lang/en.json:377]: hh = 12h
HH = 24h
a = am/pm

Comment (MrKrisKrisu) [reply resources/lang/en.json:377]: Oh, wait.

Comment (MrKrisKrisu) [reply resources/lang/en.json:377]: Line 375 fixed. 💚

**Review by HerrLevin — APPROVED** [review]

**Pull request merged**